components/pages/Questions.tsx

```
@@ -159,21 +183,27 @@ const Questions = memo(function Questions() {
+      // Add delay for micro-interaction
+      await new Promise(resolve => setTimeout(resolve, 400));
+      
       try {
         await saveAnswer(questionId.toString(), option);
         if (questionId < questions.length) {
           setCurrentQuestion(questionId + 1);
         } else {
           setShowBaseSpirits(true);
         }
+        setSelectedOption(null); // Reset selection
       } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)